sourcecode/hub/resources/views/content/show.blade.php

```diff
@@ -1,27 +1,142 @@
 <x-sidebar-layout>
     <x-slot:title>{{ $content->latestPublishedVersion->resource->title }}</x-slot:title>
 
+    <div class="container mt-1">
+        <div class="row">
+            <div class="col-6">
+                {{ trans('messages.created')}}: {{ $content->created_at->format('M d, Y') }} by {{ $authorName }}
+            </div>
+        </div>
+
+        <div class="row mt-1">
+            <div class="col-12">
+                {{ trans('messages.last-updated')}}: {{ $content->updated_at->format('M d, Y') }}
+            </div>
+        </div>
+    </div>
+
     <x-slot:head>
         <x-oembed-links />
     </x-slot:head>
 
-    <x-lti-launch :launch="$launch" />
+    <div class="container">
+        <div class="row justify-content-center">
+            <div class="row justify-content-start">
+                <x-lti-launch :launch="$launch" />
+            </div>
+        </div>
+        <div class="row justify-content-center mt-4">
+            <div class="buttons-container d-flex justify-content-between w-100">
+                <div>
+                    <button type="button" class="btn btn-secondary" aria-label="{{ trans('messages.edit')}}">
+                        {{ trans('messages.edit')}}
+                        <x-icon name="pencil" class="me-1" />
+                    </button>
+                    <button type="button" class="btn btn-secondary" aria-label="{{ trans('messages.use-resource')}}">
```

Comment:
The content of the element is a readable text, no need to repeat it using `aria-label`